genpieces.py

```diff
@@ -212,7 +212,7 @@ def generate_tile_2_lines(a, b, c, d, e, f, g, h, scale=1,
 
 
 def open_file(datapath, filename):
-    return file(os.path.join(datapath, filename), "w")
+    return open(os.path.join(datapath, filename), "w")
```

Comment:
You can use `open` without importing it from `io`.

Reply:
I agree that open() is a built-in in both python 2 and python 3. In python 2, io.open() was different from open(), however the differences have been removed in python 3. I added that just out of habit. Thanks for noticing, I will remove it in the next commit.

Reply:
Done in commit `766b774`